src/client/app/needs/new_need_register.html

```diff
@@ -122,25 +88,22 @@ <h2 class="text-left">Нова потреба</h2>
                 </div>
             </div>
             <div class="form-group row">
-                <label class="control-label col-sm-2">Актуальність</label>
-                <div class="col-sm-3">
-                    <input type="date" value="2015-01-01" max="2018-01-01" min="2015-01-01"> <!-- tbd, working only in Chrome -->
-                </div>
+                <date-picker date="vm.dt"></date-picker>
             </div>
             <div class="form-group row">
                 <label class="control-label col-sm-2">Зручний час</label>
                 <div class="col-sm-3">
-                    <input type="text" class="form-control" id="s_time" placeholder="Пн-Пт, 10:00 - 19:00;">
+                    <input type="text" class="form-control" id="s_time" ng-model="vm.need.suitTime" placeholder="Пн-Пт, 10:00 - 19:00;">
```

Comment:
rename to suitable time.
it's not obvious that this is suitable time.
It'll help someone who'll read the code after you.